homeassistant/components/camera/__init__.py

```diff
@@ -109,12 +109,13 @@ class Image:
 
     async with async_timeout.timeout(10):
         source = await camera.stream_source()
+        protocol = await camera.stream_protocol()
```

Comment:
We shouldn't add a new method. Instead, `stream_source` should return a StreamInfo object that can contain more information than just source.